storybook/cypress/e2e/ozon-content.cy.ts

```diff
@@ -443,7 +443,10 @@ describe("Ozon Content", () => {
       .find(".editaction-label")
       .and("have.text", "Toegevoegd:");
 
-    // cy.percySnapshot();
+    cy.get("dso-ozon-content.hydrated")
+      .shadow()
+      .find("dso-image-overlay.hydrated")
+      .then(() => cy.get("dso-ozon-content.hydrated").matchImageSnapshot());
```

Comment:
De `then()` is niet nodig als je de argumenten uit de callback functie niet gebruikt.

```suggestion
      .should('exist')
      .get("dso-ozon-content.hydrated").matchImageSnapshot();
```